changes/566.feature.md

```diff
@@ -1 +1 @@
-Using python-dotenv for auto detect .env file
+Can add environmental variables through files that were previously set as a bash file
```

Comment:
Now it is much better in the sense of expressing the intention, but still there are more improvements required.

First, "Can add ~~" is not an imperative form but an incomplete fragment.

Second, since the changelog should include technical details to make other developers aware of, when introducing new dependencies and the new dependency takes a key role like this PR, I believe it's better to mention `python-dotenv` explicitly.